### PR TITLE
Task/restore authenticated user

### PR DESCRIPTION
**Pull request by nmcmahon-unosquare**

### CODE REVIEW

File: v2/AdminCore.Services/AuthenticatedUser.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    public AuthenticatedUser(IHttpContextAccessor httpContextAccessor, IEmployeeService employeeService)
+    {
+      _httpContentAccessor = httpContextAccessor;
+      _employeeService = employeeService;
+    }
+
+    public EmployeeDto RetrieveLoggedInUser()
+    {
+      var userDetails = GetLoggedInUserDetails();
```
Comment (BrownMark) [inline v2/AdminCore.Services/AuthenticatedUser.cs:24]: Might be worth doing 2 things here:
1. Checking does that key exist in the dictionary.
2. store the key itself in a constants file (along with any other details we may query from the dictionary).

**Review by BrownMark — CHANGES_REQUESTED** [review]
Looks good, just minor suggestion

File: v2/AdminCore.Services/AuthenticatedUser.cs (hunk 1 above)
Comment (nmcmahon-unosquare) [reply v2/AdminCore.Services/AuthenticatedUser.cs:24]: Done

**Comment by nmcmahon-unosquare** [comment]
Closing as new azure approach being used.

**Pull request closed without merging**